### PR TITLE
Review crud

### PR DESCRIPTION
**Pull request by md1guy**

### CODE REVIEW

File: src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewServiceImpl implements ReviewService {
+
+    @Autowired
+    ModelMapper modelMapper;
+
+    @Autowired
+    ReviewRepository reviewRepository;
```
Comment (Pitcity) [inline src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java:19]: Great one, but check in which cases its better to use autowiring through constructor whether then like a field (intellij might hint you that ;)

File: src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public Review addReview(Review review) {
+
+        return reviewRepository.save(review);
+    }
+
+    @Override
+    public Review getReviewById(Long reviewId) {
+
+        if (reviewRepository.findById(reviewId).isPresent()) {
+            return reviewRepository.findById(reviewId).get();
```
Comment (Pitcity) [inline src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java]: "reviewRepository.findById(reviewId)" calling DB 2 times (line28,29) for the same purpose

File: src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    @Override
+    public Review updateReviewById(Review review, Long reviewId) {
+
+        if (reviewRepository.findById(reviewId).isPresent()) {
+            review.setId(reviewId);
+            return reviewRepository.save(review);
+        } else {
+            return null;
```
Comment (Pitcity) [inline src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java:44]: all those "return null" are a bit confusing, how to know from the client side that something went wrong?

read about ControllerAdvice

File: src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java (hunk 1 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java:19]: I will find out, thank you.

File: src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java (hunk 3 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java:44]: We still wait for exceptions mechanism implementation, I think, we'll rework this correctly today.

File: src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java (hunk 2 above)
Comment (md1guy) [reply src/main/java/com/softserve/actent/service/impl/ReviewServiceImpl.java]: Fixed, thank you. 👍

**Review by Pitcity — APPROVED** [review]

**Review by RBarnabass — APPROVED** [review]

**Review by nazarmavko — APPROVED** [review]

**Pull request merged**